packages/devextreme/scss/widgets/material/htmlEditor/_index.scss

```diff
@@ -335,3 +336,7 @@
     background-color: color.change($base-accent, $alpha: 0.5);
   }
 }
+
+.dx-formdialog.dx-dropdowneditor-overlay.dx-popup-wrapper .dx-overlay-content {
```

Comment:
I'm sure that hierarchy structure is more readable and scaleable.

Reply:
I need a strong selector for this case.

Reply:
We need this selector because we have added .dx-dropdowneditor-overlay class. After it some styles was incorrect. This strong selector fixed it.

Reply:
I meant placing all the classes in one line. Usually, we try to use nesting for better readability.